src/interval.d.ts

```diff
@@ -1,3 +1,9 @@
+// For internal use.
+export type PeriodicTimeIntervalName =
+  | (`${number} ${TimeIntervalName}s` & Record<never, never>) // see https://github.com/microsoft/TypeScript/issues/29729
```

Comment:
```suggestion
  | ((`1 ${TimeIntervalName}` | `${number} ${TimeIntervalName}s`) & Record<never, never>) // see https://github.com/microsoft/TypeScript/issues/29729
```

"1 year" is accepted and that feels good, let's reflect it in the types?